internal/backend/dispatcher/eventsworkflow.go

```diff
@@ -24,6 +25,62 @@ func (d *dispatcher) createEventRecord(ctx context.Context, eventID sdktypes.Eve
 	return nil
 }
 
+func (d *dispatcher) resolveEnv(ctx context.Context, env string) (envID sdktypes.EnvID, err error) {
+	if env == "" {
+		return sdktypes.InvalidEnvID, nil
+	}
+
+	parts := strings.Split(env, "/")
+	switch len(parts) {
+	case 1:
+		if sdktypes.IsEnvID(parts[0]) {
+			return sdktypes.ParseEnvID(parts[0])
+		}
```

Comment:
It's hard to follow the logic in this branch, i.e. need to skip to the end of the function. Why not call `return` here?

Also, why do we even support IDs? They're a purely internal implementation detail which users should not be exposed to or rely upon. It should be just names.

Reply:
simplified now. no ids.